data/load-seed-data.js

```diff
@@ -28,7 +28,7 @@ async function run() {
     await Promise.all(
       favorites.map(favorite => {
         return client.query(`
-                    INSERT INTO favorites (id, favorite, owner_id)
+                    INSERT INTO favorites (id, favorite_id, owner_id)
```

Comment:
I think this needs to be (favorite_id, owner_id) or (breed_id, owner_id). If I'm remembering correctly we can't set id when its type is SERIAL.